pkg/component/controller/workerconfig/reconciler_test.go

```diff
@@ -374,22 +374,22 @@ func TestReconciler_ResourceGeneration(t *testing.T) {
 	}))
 
 	configMaps := map[string]func(t *testing.T, expected *kubeletConfig){
-		"worker-config-default-1.31": func(t *testing.T, expected *kubeletConfig) {
+		"worker-config-default-1.31": func(_ *testing.T, expected *kubeletConfig) {
 			expected.CgroupsPerQOS = ptr.To(true)
 			expected.FeatureGates = map[string]bool{"kubelet-feature": true}
 		},
 
-		"worker-config-default-windows-1.31": func(t *testing.T, expected *kubeletConfig) {
+		"worker-config-default-windows-1.31": func(_ *testing.T, expected *kubeletConfig) {
 			expected.CgroupsPerQOS = ptr.To(false)
 			expected.FeatureGates = map[string]bool{"kubelet-feature": true}
 		},
 
-		"worker-config-profile_XXX-1.31": func(t *testing.T, expected *kubeletConfig) {
+		"worker-config-profile_XXX-1.31": func(_ *testing.T, expected *kubeletConfig) {
 			expected.Authentication.Anonymous.Enabled = ptr.To(true)
 			expected.FeatureGates = map[string]bool{"kubelet-feature": true}
 		},
 
-		"worker-config-profile_YYY-1.31": func(t *testing.T, expected *kubeletConfig) {
+		"worker-config-profile_YYY-1.31": func(_ *testing.T, expected *kubeletConfig) {
```

Comment:
None of  them use `testing.T`. Name `configMaps` could be improved.

Reply:
Done. PTAL.